internal/run/source.go

```diff
@@ -265,17 +261,28 @@ func (w *Workflow) RunSource(ctx context.Context, parentStep *workflowTracking.W
 		})
 	}
 
-	if !w.SkipLinting {
+	if w.SkipLinting {
+		w.OnSourceResult(sourceRes)
+	} else {
 		sourceRes.LintResult, err = w.validateDocument(ctx, rootStep, sourceID, currentDocument, rulesetToUse, w.ProjectDir)
+		fmt.Println("sourceRes.LintResult", sourceRes.LintResult)
```

Comment:
We cannot have this kind of print statement in the code
Besides being a debug log, using `fmt.Println` instead of `log.From(ctx)...` breaks the interactive output

Reply:
whoops yep, just debugging missed this thanks